tests/installation/ipxe_install.pm

```diff
@@ -47,10 +47,9 @@ sub poweron_host {
 sub set_pxe_boot {
     while (1) {
         my $stdout = ipmitool('chassis bootparam get 5');
-        last if $stdout =~ m/Force PXE/;
+        last if $stdout =~ m/Boot Flag Valid[\d\D]*Force PXE/;
         diag "setting boot device to pxe";
-        my $options;
-        $options = 'options=efiboot' if get_var('IPXE_UEFI');
+        my $options = get_var('IPXE_UEFI') ? 'options=efiboot' : '';
```

Comment:
is really needed to assign empty string?

Reply:
or you just want merge two lines in one line?

Reply:
ok clear , i udnerstand now

Reply:
the snippet in the test your provided above https://openqa.suse.de/tests/7805769/logfile?filename=autoinst-log.txt 
```
[2021-12-07T20:53:00.262931+01:00] [debug] setting boot device to pxe
Use of uninitialized value $options in concatenation (.) or string at sle/tests/installation/ipxe_install.pm line 54.
	ipxe_install::set_pxe_boot() called at sle/tests/installation/ipxe_install.pm line 141
...
```
It is not mandatory but noisy.